install/kubernetes/helm/istio/charts/kiali/values.yaml

```diff
@@ -69,6 +69,12 @@ dashboard:
   jaegerURL:  # If you have Jaeger installed and it is accessible to client browsers, then set this property to its external URL. Kiali will redirect users to this URL when Jaeger tracing is to be shown.
 prometheusAddr: http://prometheus:9090
 
+kubernetesConfig:
+  burst: 200
+  qps: 175
+  cacheEnabled: false
+  cacheDuration: 300
+
```

Comment:
AFAIK, this is not needed. It should be enough to omit these changes in this file and let the operator set these defaults.

Reply:
I've not checked Kiali operator yet. But the reason I am setting this is when the number of pods/service is large, we need to increase from the defaults otherwise `apiserver` will throttle us.

Reply:
Ops! I said the operator when I wanted to say Kiali itself. You already pointed that these values are hard-coded in https://github.com/kiali/kiali/blob/1e1b94b69e02cc582be46ad0fbacef7bf48d6053/config/config.go#L339.

So, the absence of this in helm will make Kiali to use the right defaults that you are replicating here.
If you prefer to have this as a kind of documentation for helm charts, I'd suggest you to include these lines as comments (similarly to the ldap properties).

It's only a suggestion. The decision is yours. I have no strong opinion.

Reply:
Yes, in my case, I want to *override* the defaults. So in my installation, I can customise the burst to to 1000, for example.